R/isIntersect.R

```diff
@@ -0,0 +1,42 @@
+##################################################################################
+#' Do SPatial objects intersect
+#'
+#' is Intersect verifies if the extent of two Spatial objects are intersecting. Returns TRUE if its argument intersect.
+#' This function was created to avoid empty output created mask() function in dataPrepRaster. Mask need the layers
+#'
+#' @param x   A \code{RasterLayer} object or the name of one.
+#' @param layer A RasterLayer object or a SpatialPolygonDataFrame object.
+#'
+#' @return Logical TRUE if the intersect does occur between the 2 layers.
+#'
+#' @importFrom raster extent crop raster
+#' @docType methods
+#'
+#' @author Melina Houle
+#' @examples
+#' library(raster)
+#' library(maptools)
```

Comment:
`maptools` needs to be added to Suggests if you are using it for the examples/tests. However, I'd like to remove the dependency *because* it is only providing data for examples/tests.

Only run the example if the user has `maptools` installed (it'd only be in Suggest, not Imports/Depends).